app/src/main/java/ingsw/group1/findmyphone/activity/ModifyContactActivity.java

```diff
@@ -0,0 +1,82 @@
+package ingsw.group1.findmyphone.activity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.EditText;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.eis.smslibrary.SMSPeer;
+
+import ingsw.group1.findmyphone.R;
+import ingsw.group1.findmyphone.contacts.SMSContactSwipeCallback;
+import ingsw.group1.findmyphone.contacts.SMSContact;
+import ingsw.group1.findmyphone.contacts.SMSContactManager;
+
+/**
+ * Activity for adding a new contact
+ *
+ * @author Giorgia Bortoletti
+ */
+public class ModifyContactActivity extends AppCompatActivity {
+
+    private static String contactAddress;
+
+    private EditText contactNameField;
+    private TextView contactPhoneField;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'contactPhoneField' could be replaced by a local variable.](https://app.codacy.com/manual/RicDeZen/ingsw-group1-findmyphone/pullRequest?prid=5065092)